tensorflow_datasets/structured/movielens.py

```diff
@@ -0,0 +1,422 @@
+              "(no genres listed)" is considered a label itself.
+              This dataset was generated on September 26, 2018."""),
+          version='0.1.0',
+      ),
+      tfds.core.BuilderConfig(
+          name='100k',
+          description=textwrap.dedent("""\
+              This dataset contains 100,000 ratings (1-5) from 943 users
+              on 1682 movies.
+              Ratings are in whole-star increments.
+              Each user has rated at least 20 movies.
+              This dataset is the oldest version of the MovieLens dataset.
+              The movies data and ratings data are joined on "movie id".
+              The ratings data and users data are joined on "user id".
+              Occupation labels standardized for "user_occupation_label" to
+              ensure consistency.
+              The original occupation strings are also preserved in
+              "user_occupation_string"."""),
+          version='0.1.0',
+      ),
+      tfds.core.BuilderConfig(
+          name='1m',
+          description=textwrap.dedent("""\
+              This dataset contains 1,000,209 anonymous ratings of
+              approximately 3,900 movies made by 6,040 MovieLens users
+              who joined MovieLens in 2000.
+              Ratings are in whole-star increments.
+              Each user has rated at least 20 movies.
+              This dataset is the largest dataset that includes
+              demographic data.
+              The movies data and ratings data are joined on "MovieId".
+              The ratings data and users data are joined on "UserId".
+              The age values are divided into ranges and the lowest
+              age value for each range is used in the data instead of
+              the actual values.
+              Occupation labels standardized for "user_occupation_label" to
+              ensure consistency.
+              The original occupation strings are also preserved in
+              "user_occupation_string"."""),
+          version='0.1.0',
+      ),
+      tfds.core.BuilderConfig(
+          name='20m',
+          description=textwrap.dedent("""\
+              This dataset contains contains 20000263 ratings
+              across 27278 movies.
+              These data were created by 138493 users between
+              January 09, 1995 and March 31, 2015.
+              Ratings are in half-star increments.
+              Each user has rated at least 20 movies.
+              This dataset does not contain demographic data.
+              The movies data and ratings data are joined on "movieId".
+              Genres are converted into sequences of labels.
+              "(no genres listed)" is considered a label itself.
+              This dataset was generated on October 17, 2016."""),
+          version='0.1.0',
+      ),
+  ]
+
+  VERSION = tfds.core.Version('0.1.0')
+
+  def _info(self) -> tfds.core.DatasetInfo:
+    """Return a DatasetInfo according to self.builder_config."""
```

Comment:
Nit: 'Return a DatasetInfo' -> 'Returns DatasetInfo'

Reply:
Fixed in 333dc4fec316388b6440cb7baf64b755905c724e.